### PR TITLE
CM-189: add search to benefit plans resolve

### PR DESCRIPTION
**Pull request by jdolkowski**
https://openimis.atlassian.net/browse/CM-189

This PR adds search field in BenefitPlan schema.

### CODE REVIEW

File: social_protection/schema.py [hunk 1]
```diff
@@ -157,6 +158,11 @@ def resolve_benefit_plan(self, info, **kwargs):
         if beneficiary_status:
             filters.append(Q(beneficiary__status=beneficiary_status) | Q(groupbeneficiary__status=beneficiary_status))
 
+        search = kwargs.get("search", None)
```
Comment (sniedzielski) [inline social_protection/schema.py:161]: Why we define addtional 'search' filter? I think we can use directly from default graphQL schema for BenefitPlan. (definied by default name and code)

**Review by sniedzielski — COMMENTED** [review]
I left a comment, please refer to this comment

File: social_protection/schema.py (hunk 1 above)
Comment (jdolkowski) [reply social_protection/schema.py:161]: You are right, I was doing it analogically to the Product picker and didn't put much thought into that.

Comment (sniedzielski) [reply social_protection/schema.py:161]: In that case please fix your solution in fe modules to use filters provided by graphQL schema by default (from ORM)

**Comment by sniedzielski** [comment]
Ticket closed due to changing approach of filtering BenefitPlan in picker.

**Comment by sniedzielski** [comment]
Ticket closed due to changing approach of filtering BenefitPlan in picker.

**Pull request closed without merging**